.env.example

```diff
@@ -1 +1,29 @@
-REDIS_PASSWORD=
\ No newline at end of file
+RPC_URLS=https://arbitrum-sepolia-rpc.publicnode.com,https://arbitrum-sepolia.drpc.org,https://sepolia-rollup.arbitrum.io/rpc,https://public.stackup.sh/api/v1/node/arbitrum-sepolia
+REDIS_PASSWORD=
+REDIS_HOST=redis-server
+MULTICALL_BATCH_SIZE=300
+FORK_RPC_URL=https://arbitrum-sepolia-rpc.publicnode.com
+PRIVATE_KEYS_1=
+PRIVATE_KEYS_2=
+RPC_FIRST_PROBABILITY=0.75
+POSITIONS_PER_RUN_ETH_DOMINANCE=15000
+POSITIONS_PER_RUN_BTC_DOMINANCE=10000
+POSITIONS_PER_RUN_QUANTUM_CATS=3000
+POSITIONS_PER_RUN_CS2_SKINS=6000
+POSITIONS_PER_RUN_BITCOIN_FROGS=5000
+POSITIONS_PER_RUN_INK=5000
+POSITIONS_PER_RUN_NODEMONKES=5000
+WORKERS_AMOUNT_ETH_DOMINANCE=3
+WORKERS_AMOUNT_BTC_DOMINANCE=3
+WORKERS_AMOUNT_QUANTUM_CATS=1
+WORKERS_AMOUNT_CS2_SKINS=1
+WORKERS_AMOUNT_BITCOIN_FROGS=1
+WORKERS_AMOUNT_INK=1
+WORKERS_AMOUNT_NODEMONKES=1
+CRON_SCHEDULE_ETH_DOMINANCE=5,35 * * * * *
+CRON_SCHEDULE_BTC_DOMINANCE=9,39 * * * * *
+CRON_SCHEDULE_QUANTUM_CATS=13,43 * * * * *
+CRON_SCHEDULE_CS2_SKINS=17,47 * * * * *
+CRON_SCHEDULE_BITCOIN_FROGS=21,51 * * * * *
+CRON_SCHEDULE_INK=25,55 * * * * *
+CRON_SCHEDULE_NODEMONKES=29,59 * * * * *
```

Comment:
I would use config files instead of the `.env` for this kind of things. That way, we can give more structure to it (eg. nested objects in a `.ts` file) and use the appropriate types instead of everything being a string that we have to then parse.

Reply:
Done